# Down track distance in meters
down_track: 50.0

# Uptrack distance in meters
up_track: 50.0

# Minimum gap distance to maintain between vehicles in meters
min_gap: 4.0

# Sender id in string
sender_id: 'USDOT-49096'

# Event reason in string
event_reason: 'MOVE OVER LAW'

# Event type in string
event_type: 'CLOSED'

# Start/End marker data timeout in seconds from the UI
# Also, this amount will to take to switch from using markers' data to using TORC Pinpoint data
# Note: It was found that the UI publishing of the start and end markers slows to ~ 1 Hz when the
# user switches to a different tab in the browser, so this should at least be more than 1 second
geofence_start_end_data_timeout: 2.0
